BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -89,19 +89,22 @@ public class KeyringStore: ObservableObject {
   }
 
   private let keyringService: BraveWalletKeyringService
+  private let walletService: BraveWalletBraveWalletService
   private var cancellable: AnyCancellable?
   private let keychain: KeychainType
 
   public init(
     keyringService: BraveWalletKeyringService,
+    walletService: BraveWalletBraveWalletService,
     keychain: KeychainType = Keychain()
   ) {
     self.keyringService = keyringService
+    self.walletService = walletService
     self.keychain = keychain
     
     self.keyringService.add(self)
     updateKeyringInfo()
-    self.keyringService.defaultKeyringInfo { [self] keyringInfo in
+    self.keyringService.keyringInfo(BraveWallet.DefaultKeyringId) { [self] keyringInfo in
```

Comment:
Shouldn't we get the selected coin and use the coin's `keyringId` here like in `updateKeyringInfo()` below?

Reply:
Or maybe this is similar to `SettingsStore` checking if a keyring is setup, which we initially setup with ethereum.

Reply:
yep :)